pom.xml

```diff
@@ -56,27 +56,24 @@
 		<!-- TODO: Arrange the dependencies and variables in alphabetical order -->
 		<!-- TODO: Remove dependencies on API where omods are available -->
 		<adminuiVersion>1.2.3</adminuiVersion>
-		<addresshierarchyVersion>2.9</addresshierarchyVersion>
+		<addresshierarchyVersion>2.11.0</addresshierarchyVersion>
 		<allergyuiVersion>1.6</allergyuiVersion>
 		<appframeworkVersion>2.11.0</appframeworkVersion>
-		<appuiVersion>1.8.0</appuiVersion>
+		<appuiVersion>1.9.0</appuiVersion>
 		<appointmentschedulingVersion>1.7.0</appointmentschedulingVersion>
-		<appointmentschedulinguiVersion>1.5.0</appointmentschedulinguiVersion>
-		<atlasVersion>2.2</atlasVersion>
 		<calculationVersion>1.2</calculationVersion>
-		<chartsearchVersion>2.1.0</chartsearchVersion>
 		<coreappsVersion>1.20.0-SNAPSHOT</coreappsVersion>
```

Comment:
@dbaluku Upgrade coreapps to the latest snapshot 1.21.0-SNAPSHOT

Reply:
Done